### ISSUE TITLE
MacOS Save Dialog - Buttons too small

### ISSUE THREAD

**Issue by maxwxyz** [obelisk79/OpenTheme#6]
Buttons on macOS save dialog when closing with unsaved changes are too small on light and dark:
![grafik](https://github.com/obelisk79/OpenTheme/assets/6246609/040ef9c7-3fec-40ae-aa7c-3b8aa8924300)

**Comment by jacprog** [comment]
Works for me on OpenLight
<img width="447" alt="save Screenshot 2024-05-12 at 8 19 14 AM" src="https://github.com/obelisk79/OpenTheme/assets/23371617/a6fe7ce2-c663-49e8-bea2-58ac00d30756">
OS: macOS 14.4.1
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.36958 (Git)
Build type: Release
Branch: main
Hash: a041129090b8dcbb8367636840d3c668ef30a933
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: C/Default (C)
Installed mods: 
  * freecad.gears-master 1.2.0
  * FreeCAD_Assembly4-master 0.50.7
  * OpenTheme 2024.5.3
  * freecad-slic3r-tools-master
  * OpenLight_Beta_20240208 2024.2.7
  * FreeCAD_FastenersWB-master 0.5.1

**Comment by maxwxyz** [comment]
maybe because the `Apply answer to all` checkbox is missing on your side?

**Comment by obelisk79** [comment]
> maybe because the `Apply answer to all` checkbox is missing on your side?

Why do you have Apply answer to all in your dialog bot not the other guy? It is quite possible that you are right about that being the cause of the buttons being so small.

**Comment by obelisk79** [comment]
@maxwxyz is this still and issue for you?

**Comment by maxwxyz** [comment]
Yes, the `apply answer to all` is showing, when you have multiple unsaved open documents.
